### PR TITLE
LC-2600-Integrate lpg-management with csl-identity service for login-2

### PR DESCRIPTION
**Pull request by neeraj-jain-cbo**

### CODE REVIEW

File: src/identity/auth.ts [hunk 1]
```diff
@@ -51,11 +51,12 @@ export class Auth {
 		let strategy: oauth2.Strategy
 		strategy = new oauth2.Strategy(
 			{
-				authorizationURL: `${this.config.authenticationServiceUrl}/oauth/authorize`,
+				authorizationURL: this.config.authorizationPath,
```
Comment (jaylco) [inline src/identity/auth.ts:54]: This should be ``authorizationURL: `${this.config.authenticationServiceUrl}${this.config.authorizationPath}` `` I think

File: src/identity/auth.ts [hunk 2]
```diff
@@ -51,11 +51,12 @@ export class Auth {
...
 				callbackURL: `${this.config.callbackUrl}/authenticate`,
 				clientID: this.config.clientId,
 				clientSecret: this.config.clientSecret,
-				tokenURL: `${this.config.authenticationServiceUrl}/oauth/token`,
+				tokenURL: this.config.authTokenPath
```
Comment (jaylco) [inline src/identity/auth.ts:58]: This should be ``authorizationURL: `${this.config.authenticationServiceUrl}${this.config.authTokenPath}` ``

File: src/identity/auth.ts (hunk 1 above)
Comment (neeraj-jain-cbo) [reply src/identity/auth.ts]: Comment applied.

File: src/identity/auth.ts (hunk 2 above)
Comment (neeraj-jain-cbo) [reply src/identity/auth.ts]: Comment applied

**Review by jaylco — APPROVED** [review]

**Pull request merged**